### PR TITLE
Drop "manual build" for "Package Manager (Bower/npm)"

### PR DESCRIPTION
**Pull request by notlee**
Manual Build doesn't mean anything outside Origami. The choice is
really between the Origami Build Service or installing components with
a package manager (which requires a build step).
https://financialtimes.slack.com/archives/CSW6B2VAN/p1617986490072400

now:
Screenshot 2021-04-16 at 10 37 55

before:
Screenshot 2021-04-16 at 10 37 51

### CODE REVIEW

File: _tutorials/manual-build.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ redirect_from:
 
 # {{page.title}}
 
-Adding Origami components to your product through the manual build process gives you more granular control over their styling and their behaviour. It requires more set up though, so we're providing an in-depth walkthrough for building a page for an article about fruit.
+Installing Origami components with a package manager (manual build) gives you more granular control over their styling and their behaviour within your project. It requires more set up though, so we're providing an in-depth walkthrough for building a page for an article about fruit.
```
Comment (notlee) [inline _tutorials/manual-build.md:13]: I've left some recognition of our previous terminology for the time being, and have left the old url intact.

>with a package manager (manual build)

File: _components/initialising.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Initialising a component
-description: Documentation on how to initialise Origami components when you're using a manual build process.
+description: Documentation on how to initialise Origami components when you're install a component with a package manager (Bower/npm).
```
Comment (JakeChampion) [inline _components/initialising.md:3]:
```suggestion
description: Documentation on how to initialise Origami components when you're installing a component with a package manager (Bower/npm).
```

**Review by JakeChampion — APPROVED** [review]
This is good 👍 
I noticed one potential grammar(?) issue which I've added a "suggestion comment" to

File: _tutorials/manual-build.md (hunk 1 above)
Comment (JakeChampion) [reply _tutorials/manual-build.md]: Nice idea to keep manual build mentioned somewhere in the page ☺️

**Comment by notlee** [comment]
Are you happy with this change in terminology @chee?

**Comment by notlee** [comment]
Chee says yes

**Pull request merged**